src/templates/plugins/prismic.js

```diff
@@ -65,7 +71,7 @@ export default async (context, inject) => {
           return PrismicDOM.Date(date)
         }
       },
-      <% if (options.preview) { %>
+      <% if (!(process.client && process.static) && options.preview) { %>
```

Comment:
```suggestion
      <% if (!isClientStatic && options.preview) { %>
```

Reply:
Nice catch!

Reply:
Doesn't this kill the preview behavior once in production though?

Reply:
Definitely. See my `fullStatic` proposal below.